dbt/profiles.yml

```diff
@@ -11,6 +11,7 @@ athena:
       schema: z_static_unused-dbt-stub-database
       # "database" here corresponds to a Glue data catalog
       database: awsdatacatalog
+      spark_work_group: Spark_primary
```

Comment:
This is the only setting needed to actually enable the spark integration. I'll make a separate PR to capture the new workgroup, permissions, and IAM roles.